setup.py

```diff
@@ -49,7 +49,7 @@
             'flake8',
         ],
         coverage=[
-            'pytest-cov',
+            'coverage',
```

Comment:
But this is a question of discussion. If we decide to use pytest-cov instead, I would refactor all tox.ini's and change coverage to pytest-cov.